getting-started/proc-add-custom-app-file-openshift-operator.adoc

```diff
@@ -0,0 +1,101 @@
+[id='proc-add-custom-app-file-openshift-operator_{context}']
+= Adding a custom application configuration file to Red Hat OpenShift
+
+To access the {product}, you must add a custom application configuration file to OpenShift. In OpenShift, you can use the following content as a base template to create ConfigMap named `app-config-rhdh`:
+
+[source]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: app-config-rhdh
+data:
+  app-config-rhdh.yaml: |
+    app:
+      title: Red Hat Developer Hub
+
+    backend:
+      auth:
+        keys:
+        - secret: “${BACKEND_SECRET}”
+       baseUrl: https://backstage-<CUSTOM_RESOURCE_NAME>-<NAMESPACE_NAME>.<OPENSHIFT_INGRESS_DOMAIN>
+       cors:
+         origin: https://backstage-<CUSTOM_RESOURCE_NAME>-<NAMESPACE_NAME>.<OPENSHIFT_INGRESS_DOMAIN>
+----
+
+.Example
+[source]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: app-config-rhdh
+data:
+  "app-config-rhdh.yaml": |
+    app:
+      title: Red Hat Developer Hub
+      baseUrl: https://backstage-backstage-sample-my-ns.apps.ci-ln-vtkzr22-72292.origin-ci-int-gce.dev.rhcloud.com
+    backend:
+      auth:
+        keys:
+          - secret: "${BACKEND_SECRET}"
+      baseUrl: https://backstage-backstage-sample-my-ns.apps.ci-ln-vtkzr22-72292.origin-ci-int-gce.dev.rhcloud.com
+      cors:
+        origin: https://backstage-backstage-sample-my-ns.apps.ci-ln-vtkzr22-72292.origin-ci-int-gce.dev.rhcloud.com
+----
+
+[NOTE]
+====
+* There is a mandatory Backend Auth Key for {product-short}. This will reference an environment variable backed by an OpenShift Secret.
```

Comment:
"backed by" => 'defined in"

Reply:
@jianrongzhang89 Could you use suggestions, so it is easier to apply the change if needed?

Reply:
Done